state/src/tui.rs

```diff
@@ -9,6 +7,7 @@ use crossterm::{
 
 use futures::executor::block_on;
 use itertools::Itertools;
+use lazy_static::lazy_static;
```

Comment:
is this used anywhere?

Reply:
yes down below, it just wasn't erroring before because the feature flag was disabling its compilation